cli/user/src/test/java/io/pravega/cli/user/TestUtils.java

```diff
@@ -35,11 +38,60 @@ public static String executeCommand(String inputCommand, InteractiveConfig confi
         CommandArgs args = new CommandArgs(pc.getArgs().getArgs(), config);
         Command cmd = Command.Factory.get(pc.getComponent(), pc.getName(), args);
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        try (PrintStream ps = new PrintStream(baos, true, "UTF-8")) {
+        try (PrintStream ps = new PrintStream(baos, true, StandardCharsets.UTF_8)) {
             assert cmd != null;
             cmd.setOut(ps);
             cmd.execute();
         }
         return new String(baos.toByteArray(), StandardCharsets.UTF_8);
     }
+
+    /**
+     * Returns the relative path to `pravega/config` source directory from cli/user tests.
+     *
+     * @return the path
+     */
+    public static String pathToConfig() {
+        return "../../config/";
+    }
+
+    /**
+     * Creates a local Pravega cluster to test on using {@link ClusterWrapper}.
+     *
+     * @param authEnabled whether accessing the cluster require authentication or not.
+     * @param tlsEnabled whether accessing the cluster require TLS or not.
+     * @return A local Pravega cluster
+     */
+    public static ClusterWrapper createPravegaCluster(boolean authEnabled, boolean tlsEnabled) {
+        return ClusterWrapper.builder()
+                .authEnabled(authEnabled)
+                .tlsEnabled(tlsEnabled)
+                .tlsServerCertificatePath(pathToConfig() + SecurityConfigDefaults.TLS_SERVER_CERT_FILE_NAME)
```

Comment:
This and the next two lines are unnecessary if TLS is disabled. Why don't you call them only if TLS is enabled?

Reply:
Rearranged accordingly.